sopel/coretasks.py

```diff
@@ -31,6 +31,9 @@
 import re
 import time
 
+from scramp import ScramClient, ScramException
+from scramp.core import ClientStage
```

Comment:
Despite what I just said on your UTF8ONLY PR, I don't mind these `from` imports because the imported names are mostly self-evident. They aren't generic in the way `raises` is.

The only thing I might have done differently is `from scramp.core import ClientStage as ScramClientStage`.